templates/taxi/car_list.html

```diff
@@ -1,22 +1,31 @@
-{% extends "base.html" %}
+{% extends 'base.html' %}
 
 {% block content %}
   <h1>
-    Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
+    Car List
+    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">Create</a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    <label><input type="text" name="search_query" class="form-control" placeholder="Search by model" value="{{ request.GET.search_query }}" /></label>
```

Comment:
The search input name here is `search_query`, but the `CarListView` expects `search`. Ensure consistency between the form and view to make the search functionality work.